src/main/java/core/basesyntax/service/implementations/FruitTransaction.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.implementations;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public Operation getOperation() {
+        return this.operation;
+    }
+
+    public void setOperation(String value) {
```

Comment:
setter should not contain logic
```suggestion
    public void setOperation(Operation operation) {
```